kfs-purap/src/main/java/edu/arizona/kfs/module/purap/businessobject/CreditMemoItem.java

```diff
@@ -0,0 +1,78 @@
+package edu.arizona.kfs.module.purap.businessobject;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import org.kuali.kfs.module.purap.businessobject.AccountsPayableItemBase;
+import org.kuali.kfs.module.purap.businessobject.CreditMemoAccount;
+import org.kuali.kfs.module.purap.businessobject.PaymentRequestAccount;
+import org.kuali.kfs.module.purap.businessobject.PaymentRequestItem;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderItem;
+import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
+import org.kuali.kfs.module.purap.document.service.AccountsPayableService;
+import org.kuali.kfs.module.purap.util.ExpiredOrClosedAccountEntry;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public class CreditMemoItem extends org.kuali.kfs.module.purap.businessobject.CreditMemoItem {
+    
+    public CreditMemoItem() {
```

Comment:
I cannot see why this default constructor is needed in the edu.arizona version of this class, but it might have something to do with the fact that we are overriding one of the other class constructors here. Go ahead and leave it in.